ocean-and-coastal-research.csl

```diff
@@ -0,0 +1,175 @@
+
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-GB">
+  <!-- This style was edited with the Visual CSL Editor (https://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>Ocean and Coastal Research</title>
+    <title-short>OCR </title-short>
+    <id>http://www.zotero.org/styles/ocean-and-coastal-research</id>
+    <link href="http://www.zotero.org/styles/ocean-and-coastal-research" rel="self"/>
+    <link href="http://www.ocr-journal.org/instructions" rel="self"/>
```

Comment:
@azguth, thanks. Is http://www.ocr-journal.org/ the correct website? It doesn't seem to be active. I assume we can change this to http://www.io.usp.br/index.php/ocean-coast-res/normas-e-instrucoes.html ?

Reply:
our site is now online! http://www.ocr-journal.org/index.php/instructions-for-authors